src/test/java/HomeWork21.java

```diff
@@ -0,0 +1,56 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork21 extends BaseTest {
+    String newPlaylistName = "Sample Edited Playlist";
+
+    @Test
+    public void renamePlaylist() throws InterruptedException {
+
+
+        String updatedPlaylistSuccessMsg = "Updated playlist \"Sample Edited Playlist.\"";
+
+        //Log in to Koel
+        provideEmail("jason.fenstermaker1234@testpro.io");
+        providePassword("Testpro.io2");
+        loginToKoel();
+        Thread.sleep(2000);
+        //Double click
+        doubleClickPlaylist();
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.